bin/tachyon-mount.sh

```diff
@@ -213,6 +213,7 @@ case "${1}" in
         mount_local $1
         ;;
       workers)
+        bin=`cd "$( dirname "$0" )"; pwd`
         $LAUNCHER $bin/tachyon-workers.sh $bin/tachyon-mount.sh $1
```

Comment:
Hi @jsimsa . Now, I just changed the *bin/tachyon-mount.sh* file, that should address your need.